app/src/main/java/sdp/moneyrun/ui/map/MapActivity.java

```diff
@@ -409,7 +414,8 @@ private void initChronometer() {
             if (chronometerCounter < game_time) {
                 chronometerCounter += 1;
                 circleRadius -= shrinkingFactor;
-                initCircle();
+                if(chronometerCounter % 2 == 0)
```

Comment:
Is this so that it updates less frequently?

Reply:
Yes